static/src/javascripts/projects/common/modules/experiments/tests/prebid-us-mobile-sticky.js

```diff
@@ -0,0 +1,32 @@
+// @flow strict
+import { getCookie } from 'lib/cookies';
+import { isBreakpoint } from 'lib/detect';
+
+export const prebidUsMobileSticky: ABTest = {
+    id: 'PrebidUsMobileSticky',
+    start: '2019-06-11',
+    expiry: '2019-09-30',
+    author: 'Ioanna Kyprianou',
+    description: 'This asks prebid for a mobile sticky slot 320x50',
+    audience: 0.0,
+    audienceOffset: 0.0,
+    successMeasure:
+        'We can see an ad served by prebid for a mobile sticky slot',
+    audienceCriteria: 'n/a',
+    dataLinkNames: 'n/a',
+    idealOutcome:
+        'No significant impact to performance as well as higher ad yield',
+    canRun: () =>
+        (getCookie('GU_geo_continent') || 'OTHER').toUpperCase() === 'NA' &&
```

Comment:
Since the mobilesticky already only runs and this is a 0% test I don't think it's necessary to make this check here as well unless you're planning on testing a percentage other than 0%  before going live with it.

Reply:
I had the same thinking but eventually followed the same principle as when we have set the mobile sticky on live at AB zero percent as here: https://github.com/guardian/frontend/pull/21460/files . I though there was a reason for having both checks in mobile sticky and the AB test configuration. if you confirm AB tests are duplicated, will remove them :)

Reply:
Update: We want to be able to bypass the condition so I have removed the NA check in slot-config and just left the one here, (as we can by-pass AB tests)

Reply:
👍